frontend/packages/kubevirt-plugin/src/components/vms/menu-actions.tsx

```diff
@@ -15,17 +15,19 @@ import { cancelMigration } from '../../k8s/requests/vmim';
 import { cloneVMModal } from '../modals/clone-vm-modal';
 import { VMCDRomModal } from '../modals/cdrom-vm-modal';
 import { getVMStatus } from '../../statuses/vm/vm';
+import { isVMIPaused } from '../../selectors/vmi';
+import { unpauseVM, VMIActionType } from '../../k8s/requests/vmi/actions';
 
 type ActionArgs = {
   migration?: K8sResourceKind;
   vmi?: VMIKind;
   vmStatus?: VMMultiStatus;
 };
 
-const getVMActionMessage = (vm, action: VMActionType) => (
+const getVMActionMessage = (obj, action: VMActionType | VMIActionType) => (
```

Comment:
`getActionMessage`?

Reply:
I was considering that as well, but never did it. I'll consider this a sign. =) Done!